integration-tests/integration_tests/integration_tests/end_to_end_tests/int_asynchronous_express_messaging_pattern_tests.py

```diff
@@ -34,6 +37,8 @@ class AsynchronousExpressMessagingPatternTests(TestCase):
     def setUp(self):
         MHS_STATE_TABLE_DYNAMO_WRAPPER.clear_all_records_in_table()
         MHS_SYNC_ASYNC_TABLE_DYNAMO_WRAPPER.clear_all_records_in_table()
+        MHS_INBOUND_QUEUE.drain()
```

Comment:
(optional) What do you think about using drain also in @classmethod tearDownClass? (just in case clean-up is needed)

Reply:
There is a good argument for doing all fo the cleanup in a tearDown. Be a good citizen and cleanup after yourself rather then cleaning up after others...

Reply:
You're talking about `tearDown` but this line is in `setUp`. Am I missing something?

Reply:
Going to leave it as-is for now, but I added an open question to our Coding Standards to decide an approach in the future.

Reply:
I was thinking about draining the queue in case of any error (in classmethod tearDown).
Because then we will leave some garbage in the queue that will be waiting for the next build.